Flow.Launcher.Plugin/_sidebar.md

```diff
@@ -0,0 +1,32 @@
+# Flow.Launcher.Plugin
+- Flow.Launcher.Plugin
+    - [ActionContext](./flow.launcher.plugin.actioncontext.md)
+    - [AllowedLanguage](./flow.launcher.plugin.allowedlanguage.md)
+    - [BaseModel](./flow.launcher.plugin.basemodel.md)
+    - [FlowLauncherKeyDownEventArgs](./flow.launcher.plugin.flowlauncherkeydowneventargs.md)
+    - [FlowLauncherQueryEventArgs](./flow.launcher.plugin.flowlauncherqueryeventargs.md)
+    - [IAsyncPlugin](./flow.launcher.plugin.iasyncplugin.md)
+    - [IAsyncReloadable](./flow.launcher.plugin.iasyncreloadable.md)
+    - [IContextMenu](./flow.launcher.plugin.icontextmenu.md)
+    - [IFeatures](./flow.launcher.plugin.ifeatures.md)
+    - [IPlugin](./flow.launcher.plugin.iplugin.md)
+    - [IPluginI18n](./flow.launcher.plugin.iplugini18n.md)
+    - [IPublicAPI](./flow.launcher.plugin.ipublicapi.md)
+    - [IReloadable](./flow.launcher.plugin.ireloadable.md)
+    - [IResultUpdated](./flow.launcher.plugin.iresultupdated.md)
+    - [ISavable](./flow.launcher.plugin.isavable.md)
+    - [ISettingProvider](./flow.launcher.plugin.isettingprovider.md)
+    - [PluginInitContext](./flow.launcher.plugin.plugininitcontext.md)
+    - [PluginMetadata](./flow.launcher.plugin.pluginmetadata.md)
+    - [PluginPair](./flow.launcher.plugin.pluginpair.md)
+    - [Query](./flow.launcher.plugin.query.md)
+    - [Result](./flow.launcher.plugin.result.md)
+    - [ResultUpdatedEventArgs](./flow.launcher.plugin.resultupdatedeventargs.md)
+    - [SpecialKeyState](./flow.launcher.plugin.specialkeystate.md)
+- Flow.Launcher.Plugin.SharedCommands
+    - [FilesFolders](./flow.launcher.plugin.sharedcommands.filesfolders.md)
+    - [SearchWeb](./flow.launcher.plugin.sharedcommands.searchweb.md)
+    - [ShellCommand](./flow.launcher.plugin.sharedcommands.shellcommand.md)
+- Flow.Launcher.Plugin.SharedModels
+    - [MatchResult](./flow.launcher.plugin.sharedmodels.matchresult.md)
+    - [SearchPrecisionScore](./flow.launcher.plugin.sharedmodels.searchprecisionscore.md)
```

Comment:
How to handle this.

## build three folders

- `Flow.Launcher.Plugin`
- `Flow.Launcher.Plugin.SharedCommands`
- `Flow.Launcher.Plugin.SharedModels`

Each folder has its own `_sidebar.md`

## One folder, and hold three toc (table of content) files.

- `Flow.Launcher.Plugin.md`
- `Flow.Launcher.Plugin.SharedCommands.md`
- `Flow.Launcher.Plugin.SharedModels.md`

Let `_sidebar.md` links to these three toc file. And these toc files link to MD file.